pyproj/transformer.py

```diff
@@ -789,6 +789,22 @@ def transform(  # pylint: disable=invalid-name
         '33  98'
 
         """
+        if inplace and getattr(xx, "typecode", "") == "d":
```

Comment:
Thoughts about moving this to `_copytobuffer` instead?

Reply:
You could move it to the top of the function so it would skip everything else if true.

Reply:
Yeah, I tried that too and it unfortunately still adds quite a bit of overhead with both of those. 1.5 seconds vs 1 second on my laptop here.

Although, both of those would be "easy" immediate gains.